package students.jelena_kaverska.lesson_12.level_3.task_21;

class Task21 {
    //Какой результат мы получим?

    //1. Got the Test Exception
    //   Inside finally block
}